nginx/etc/nginx/nginx.conf.ctmpl

```diff
@@ -8,21 +8,29 @@ events {
     worker_connections  1024;
 }
 
-
 http {
     include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
 
+    map $status $isError {
+        ~^2 0;
+        default 1;
+    }
+
     log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                       '$status $body_bytes_sent "$http_referer" '
                       '"$http_user_agent" "$http_x_forwarded_for"';
 
     access_log  /var/log/nginx/access.log  main;
 
     sendfile        on;
+    #tcp_nopush     on;
```

Comment:
We do not. I simply did not prune them. I copied these files form the Nginx repo, where that line has existed since last November, and modified them to suit.